README.md

```diff
@@ -33,7 +33,7 @@ All config is in sam.[ENVIRONMENT].yml templates, encrypted as necessary.
 
 The following will invoke the lambda against the sample `event.json`
 ```
-sam local invoke --event event.json --region us-east-1 --template sam.[ENVIRONMENT].yml --profile [aws profile]
+AWS_ACCESS_KEY_ID=[your access key id] AWS_SECRET_ACCESS_KEY=[your secret access key] sam local invoke --event event.json --region us-east-1 --template sam.qa.yml --profile nypl-digital-dev
```

Comment:
Now that the env vars are in the command, you don't need the `--profile nypl-digital-dev` flag.

Reply:
True. But I also wonder why `--profile` wasn't working. I'm able to run this branch with just `--profile`. I try to keep keys out of my command history